src/mlpack/core/tree/cellbound_impl.hpp

```diff
@@ -992,8 +992,8 @@ void CellBound<MetricType, ElemType>::serialize(
     bounds = new math::RangeType<ElemType>[dim];
   }
 
-  auto boundsArray = cereal::make_array(bounds, dim);
-  ar & CEREAL_NVP(boundsArray);
+//  auto boundsArray = cereal::make_array(bounds, dim);
+//  ar & CEREAL_NVP(boundsArray);
   ar & CEREAL_NVP(minWidth);
```

Comment:
@rcurtin I have commented these lines as I have done a refactoring in `hrectbound_impl.hpp` I have changed the bound type into an `std::vector` since it will be easier to serialize, and since I can not figure the reason of using the C-Style array here.
Otherwise, everything seems to be compiled, and all tests are passing.